src/main/java/tech/autodirect/api/ApiEndpoints.java

```diff
@@ -44,9 +44,19 @@ public static void main(String[] args) {
 
 	// demo API endpoint; use for reference
 	@GetMapping("/demo")
-	public Object hello(@RequestParam String budget) {
+	public Object hello(@RequestParam String loanAmount, @RequestParam String creditScore, @RequestParam String budget,
+						@RequestParam String vehicleMake, @RequestParam String vehicleModel,
```

Comment:
Can you just hard code the vehicle info for now?
It makes more sense to just supply the loanAmount, credit score, and budget.
Just edit this file here it's fine, you can keep the concrete class unchanged.

Reply:
Yeah, no problem. @sammdu